LeaderboardBackend/Controllers/JudgementsController.cs

```diff
@@ -60,7 +63,11 @@ public async Task<ActionResult<JudgementViewModel>> GetJudgement(long id)
 	[HttpPost]
 	public async Task<ActionResult<JudgementViewModel>> CreateJudgement([FromBody] CreateJudgementRequest body)
 	{
-		User? mod = await UserService.GetUserFromClaims(HttpContext.User);
+		Guid? modId = AuthService.GetUserIdFromClaims(HttpContext.User);
+		if (modId is null)
+		{
+			return Forbid();
```

Comment:
Problem is we already have user validation done in our middleware, so this bit shouldn't be necessary.

I believe we're running into the same problem here just in a different form. Since we'd already catch and handle the null case in authZ already.

In fact I almost want to suggest we 500 here instead, since this shouldn't ever happen.

Reply:
>This shouldn't be necessary
There's no way around it with nullable reference types. The compiler can't figure out that this isn't necessary.

>Suggest we 500 here
We could throw some kind of exception, not sure what.

Reply:
After thinking about it we can keep things this way. No changes needed here